DataFormats/PatCandidates/src/PATObject.cc

```diff
@@ -0,0 +1,13 @@
+#include "DataFormats/PatCandidates/interface/PATObject.h"
+
+namespace pat {
+  const reco::CandidatePtrVector& get_empty_cpv() {
+    static reco::CandidatePtrVector EMPTY_CPV;
+    return EMPTY_CPV;
+  }
+
+  const std::string& get_empty_cpv() {
+    static std::string EMPTY_STR;
```

Comment:
```suggestion
    static const std::string EMPTY_STR;
```

Reply:
The const is needed else our additions to the static analyzer will flag this as being thread-unsafe.